packages/shared/src/utils/date/date-time.ts

```diff
@@ -3,53 +3,54 @@ import 'moment/min/locales';
 
 // Disables moment's fallback to native Date object
 // moment will return `Invalid Date` if date cannot be parsed
-moment.createFromInputFallback = function (config) {
+moment.createFromInputFallback = function (config: { _d: Date }) {
```

Comment:
I got this error: **Property 'createFromInputFallback' does not exist on type 'typeof moment'.ts(2339)**
I couldn't find any solution for it yet!

Reply:
@niloo-fs all you need is extent the type of moment here 🙂:
```
type TExtendedMoment = typeof moment & {
    createFromInputFallback: (config: { _d: Date }) => void;
};
```
```suggestion
(moment as TExtendedMoment).createFromInputFallback = function (config) {
```

Reply:
Since the "@types/moment" lib is deprecated I think it's better to extend the type!